_episodes/03-getting_started.md

```diff
@@ -82,25 +82,47 @@ Now we are ready to press `Create Project`.
 ## Play with the package
 
 Believe it or not, this package is ready to be installed.
-Just go to the upper right corner and press `Build > Install and Restart`.
+Just go to the upper right corner and press `Build > Install and Restart`. # TODO: not sure this has changed? In my console, I see `Build > Install`.
 
 ![Install and restart](../fig/install-and-restart.gif)
 
-This will install and attach the package.
-Now, you can use `hello`, the only function that this package contains so far.
-If you try it by using:
-
-~~~r
-hello()
-~~~
-{: .source}
-
-it should print:
-
-~~~r
-# Hello, world!
-~~~
-{: .output}
+This will install and attach the package. The package contains only one function so far: `hello()`. 
+
+> ## Try out the `hello()` function. Edit it, and reload the package.
+> 1. What does it print? 
+> 2. Change this function to generate a different output. (Have fun with it, perhaps add an argument or two!)
+> 3. Build and install the package, and call the function again to see that it works differently.
+> > ## Solution 
+> >    ~~~r
+> >    mysterycoffee::hello() 
+> >    ~~~
+> >    {: .source}
+> > 
+> >    ~~~r
+> >    # Hello, world!
+> >    ~~~
+> >    {: .output}
+> > 
+> >    We can add an argument like this:
+> >    ~~~r
+> >    hello <- function(name) {
+> >        print(paste0("Hello, ", name, "!"))
+> >    }
+> >    ~~~
+> >    {: .source}
+> > 
+> >    To use the updated function available in RStudio, we can use either 
+> >    ~~~r
+> >    devtools::load_all() # then call with hello("Luke")
+> >    devtools::install() # then call with mysterycoffee::hello("Luke")
+> >    ~~~
+> >    {: .source}
+> > 
+> >    Alternatively, we can also use the graphical user interface as above: `Build > Install and Restart`.
+> > 
+> >    *Note*: `load_all()` simulates installing and attaching the package. In larger projects, it allows us to iterate more quickly between changing functions and testing them in the console. 
```

Comment:
I didn't know this, and I'm not sure I completely follow. Can you elaborate on this a bit? What is the difference between Build>install; devtools::load_all and devtools::install?

Instead of putting this information in the solution to the exercise, we should mention these devtools commands earlier (next to the GUI option of Build > install). Additionally, instead of mentioning several and explaining their (subtle) differences, it could also be an option to be opinionated and just recommend one command.